docs/source/services/timelock_service/reverse-migration.rst

```diff
@@ -0,0 +1,140 @@
+.. _timelock-reverse-migration:
+
+Reverse Migration
+=================
+
+.. danger::
+
+   AtlasDB does not natively support reverse migrations. Improperly executing reverse migration from external timestamp
```

Comment:
"Natively support" seems woolly to me. What does this mean exactly?

My understanding is that we don't want people to run reverse migrations without looping one of us in?